src/arch/pmp.rs

```diff
@@ -228,7 +228,11 @@ impl PmpGroup {
             permissions < 8,
             "Permissions should not set NAPOT or TOP bits"
         );
-        self.set(idx, build_napot(from, to).unwrap(), permissions | NAPOT);
+
+        match build_napot(from, to) {
+            Some(value) => self.set(idx, value, permissions | NAPOT),
+            None => panic!("Invalid arguments for build_napot"),
+        }
```

Comment:
Out of curiosity why did you need to change that one? It seems semantically equivalent, was Kani complaining about it?

Reply:
They are equivalent, I believed Kani complained about that but it was because I was giving an hardcoded illegal value in the kani verify function. You can revert if you want